frontend/src/store/modules/sheet.ts

```diff
@@ -279,11 +274,14 @@ const actions = {
       })
     ).data;
 
-    const newSheet = convertSheet(result.data, result.included, rootGetters);
+    const sheet = convertSheet(result.data, result.included, rootGetters);
 
-    dispatch("fetchSheetList");
```

Comment:
We need to re-fetch the sheet list when patching the sheet, or we need to re-fetch the sheet list when navigating to the `sheets` tab.

Reply:
Because we have the patched sheet data, so I'd like to update it in store instead of fetching all data. Is there any bug about this code?

Reply:
the sheets store seems not reactive

Reply:
I have fixed the `SET_SHEET_BY_ID` action not reactive.